tests/gui/test_orb.py

```diff
@@ -0,0 +1,208 @@
+    assert False is store.get_state().move_touch_to_timer
+    store.dispatch(Action(Actions.MOVE_TOUCH_TO_TIMER, True))
+    assert True is store.get_state().move_touch_to_timer
+
+
+def test_store_move_timer_to_exit():
+    state = State(True, True, True)
+    store = Store(state)
+    store.dispatch(Action(Actions.MOVE_TIMER_TO_EXIT, False))
+    assert False is store.get_state().move_timer_to_exit
+    store.dispatch(Action(Actions.MOVE_TIMER_TO_EXIT, True))
+    assert True is store.get_state().move_timer_to_exit
+
+
+def test_store_touch_game_timer_stops():
+    state = State(True, True, True)
+    store = Store(state)
+    store.dispatch(Action(Actions.TOUCH_GAME_TIMER_STOPS, False))
+    assert False is store.get_state().touch_game_timer_stops
+    store.dispatch(Action(Actions.TOUCH_GAME_TIMER_STOPS, True))
+    assert True is store.get_state().touch_game_timer_stops
+
+
+def create_move_touch_to_timer_rom(data: bytearray) -> Rom:
+    rom = Rom(bytearray([0] * 0x50000))
+    rom.write(0x6913 - 4, bytearray([0xb5, 0x9a, 0xf0, 0x0e]))
+    rom.write(0x6913, data)
+    rom.write(0x6913 + len(data), bytearray([0x20, 0x12, 0xc4, 0xd0]))
+    return rom
+
+
+def test_rom_interface_read_state_move_touch_to_timer_false():
+    rom = create_move_touch_to_timer_rom(bytearray([0x8d, 0xf4, 0x7c]))
+    rom_interface = RomInterface(rom)
+    assert False is rom_interface.read_state().move_touch_to_timer
+
+
+def test_rom_interface_read_state_move_touch_to_timer_true():
+    rom = create_move_touch_to_timer_rom(bytearray([0xEA, 0xEA, 0xEA]))
+    rom_interface = RomInterface(rom)
+    assert True is rom_interface.read_state().move_touch_to_timer
+
+
+def test_rom_interface_read_state_move_touch_to_timer_invalid():
+    rom = create_move_touch_to_timer_rom(bytearray([0x11, 0x22, 0x33]))
+    rom_interface = RomInterface(rom)
+    assert None is rom_interface.read_state().move_touch_to_timer
+
+
+def test_rom_interface_write_state_move_touch_to_timer_false():
+    rom = create_move_touch_to_timer_rom(bytearray([0xEA, 0xEA, 0xEA]))
+    rom_interface = RomInterface(rom)
+    assert True is rom_interface.read_state().move_touch_to_timer
+    rom_interface.write_state(State(False, True, True))
+    assert False is rom_interface.read_state().move_touch_to_timer
+
+
+def test_rom_interface_write_state_move_touch_to_timer_true():
+    rom = create_move_touch_to_timer_rom(bytearray([0x8d, 0xf4, 0x7c]))
+    rom_interface = RomInterface(rom)
+    assert False is rom_interface.read_state().move_touch_to_timer
+    rom_interface.write_state(State(True, False, False))
+    assert True is rom_interface.read_state().move_touch_to_timer
+
+
+def test_rom_interface_write_state_move_touch_to_timer_invalid():
+    rom = create_move_touch_to_timer_rom(bytearray([0x8d, 0xf4, 0x7c]))
+    rom_interface = RomInterface(rom)
+    assert False is rom_interface.read_state().move_touch_to_timer
+    rom_interface.write_state(State(None, False, False))
```

Comment:
Either State should use the Optional keyword or None should not be considered as a valid state.